src/components/Common/Components/ProductCard/ProductCard.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import "./product-card.css";
+import laptop from "../../../../assests/img/apple.jpg";
+const ProductCard = (props) => {
```

Comment:
You can handle the logic of out of stock and Best Seller badges here itself. I can't see where you've handled this.